mkt/developers/views.py

```diff
@@ -1173,3 +1176,30 @@ def verify_data(self, data):
             return False
 
         return True
+
+
+class ContentRatingsPingbackV2(CORSMixin, UpdateModelMixin, GenericAPIView):
+    """Pingback API for IARC v2.
+
+    Should conform to the PushCert API spec. Assumes that the RatingList is
+    always sent with PushCert, so we don't need to call SearchCert afterwards
```

Comment:
we used to do SearchCert to verify the data is correct for security. i don't think it's a huge deal, but is the UUID enough?